ngraph/core/include/ngraph/op/non_max_suppression.hpp

```diff
@@ -235,6 +235,159 @@ namespace ngraph
                     clone_with_new_inputs(const OutputVector& new_args) const override;
             };
         } // namespace v4
+
+        namespace v5
+        {
+            /// \brief NonMaxSuppression operation
+            ///
+            class NGRAPH_API NonMaxSuppression : public Op
+            {
+            public:
+                enum class BoxEncodingType
+                {
+                    CORNER,
+                    CENTER
+                };
+
+                static constexpr NodeTypeInfo type_info{"NonMaxSuppression", 5};
```

Comment:
Use `NGRAPH_RTTI_DECLARATION` instead of this code.

Reply:
Fixed.